lib/default_weekday_labels_row.dart

```diff
@@ -1,18 +1,62 @@
 import 'package:flutter/widgets.dart';
+import 'package:intl/intl.dart';
 
 class CalendarroWeekdayLabelsView extends StatelessWidget {
+  final DateTime monday = DateTime(2020, 01, 06);
```

Comment:
Hey, I like the idea :) but I don't like hardcoding specific dates. Is there no other way to do it ?

Reply:
Yes, I agree that it's not ideal but unfortunately I don't know another way to make it works